RecoJets/JetProducers/plugins/BetaStarVarProducer.cc

```diff
@@ -0,0 +1,195 @@
+// -*- C++ -*-
+//
+// Package:    PhysicsTools/NanoAOD
+// Class:      BetaStarVarProducer
+// 
+/**\class BetaStarVarProducer BetaStarVarProducer.cc PhysicsTools/NanoAOD/plugins/BetaStarVarProducer.cc
+
+ Description: This produces value maps to store CHS-related variables for JERC. 
+              This includes the charged hadrons associated to CHS jets, 
+              and those associated to PU that are within the CHS jet. 
+
+ Implementation:
+     This uses a ValueMap producer functionality, and 
+     loops over the input candidates (usually PackedCandidates)
+     that are associated to each jet, counting the candidates associated
+     to the PV and those not. 
+*/
+//
+// Original Author:  Sal Rappoccio
+//
+//
+
+
+// system include files
+#include <memory>
+
+// user include files
+#include "FWCore/Framework/interface/Frameworkfwd.h"
+#include "FWCore/Framework/interface/global/EDProducer.h"
+
+#include "FWCore/Framework/interface/Event.h"
+#include "FWCore/Framework/interface/MakerMacros.h"
```

Comment:
Can you check you need this `#include`? Look at this comment https://github.com/cms-sw/cmssw/blob/c6061f4d7460bda7507c39db273be2779fdfb29e/FWCore/Framework/interface/MakerMacros.h#L7 it is a temporary solution since five years back :(

Reply:
Well, if it's a temporary five years ago, no one has fixed it, because we still need it.